web/src/message_edit.js

```diff
@@ -588,6 +588,10 @@ export function start($row, edit_box_open_callback) {
         blueslip.error("Couldn't find message ID for edit " + rows.id($row));
         return;
     }
+    
```

Comment:
The linter failure was trailing whitespace; `git show` displays it better than GitHub:

![image](https://github.com/zulip/zulip/assets/2746074/11f140e3-ec12-4d08-9ab0-a2d62c735aa2)


``` diff
diff --git a/web/src/message_edit.js b/web/src/message_edit.js
index bcfe9d8945..bc1858b9ac 100644
--- a/web/src/message_edit.js
+++ b/web/src/message_edit.js
@@ -589,6 +589,10 @@ export function start($row, edit_box_open_callback) {
         blueslip.error("Couldn't find message ID for edit", {row_id: rows.id($row)});
         return;
     }
+    
+    if ($row.find(".message_edit_form form").length !== 0) {
+        return;
+    }
 
     if (message.raw_content) {
         start_edit_with_content($row, message.raw_content, edit_box_open_callback);
```